neurostuff/openapi/neurostuff-api.v1.yml

```diff
@@ -0,0 +1,726 @@
+            group_repeated_measures_method:
+              type: string
+            hemodynamic_response_function:
+              type: string
+            high_pass_filter_method:
+              type: string
+            id:
+              type: integer
+            inclusion_exclusion_criteria:
+              type: string
+            interpolation_method:
+              type: string
+            intersubject_registration_software:
+              type: string
+            intrasubject_estimation_type:
+              type: string
+            intrasubject_model_type:
+              type: string
+            intrasubject_modeling_software:
+              type: string
+            journal_name:
+              type: string
+            length_of_trials:
+              type: string
+            modify_date:
+              type: string
+            motion_correction_interpolation:
+              type: string
+            motion_correction_metric:
+              type: string
+            motion_correction_reference:
+              type: string
+            motion_correction_software:
+              type: string
+            name:
+              type: string
+            nonlinear_transform_type:
+              type: string
+            number_of_images:
+              type: integer
+            object_image_type:
+              type: string
+            optimization_method:
+              type: string
+            order_of_preprocessing_operations:
+              type: string
+            orthogonalization_description:
+              type: string
+            owner:
+              type: integer
+            owner_name:
+              type: string
+              example: ChrisFiloGorgolewski
+            paper_url:
+              type: string
+              format: uri
+              example: 'http://dx.plos.org/10.1371/journal.pone.0118733'
+            parallel_imaging:
+              type: string
+            private:
+              type: boolean
+            pulse_sequence:
+              type: string
+            quality_control:
+              type: string
+            scanner_make:
+              type: string
+            scanner_model:
+              type: string
+            slice_timing_correction_software:
+              type: string
+            smoothing_type:
+              type: string
+            software_package:
+              type: string
+            software_version:
+              type: string
+            target_template_image:
+              type: string
+            transform_similarity_metric:
+              type: string
+            url:
+              type: string
+              format: uri
+              example: 'https://neurovault.org/collections/1039/'
+      required:
+        - '@id'
+        - '@type'
+        - analysis
+        - created_at
+        - doi
+        - name
+        - '@context'
+    analysis:
+      title: analysis
+      type: object
+      properties:
+        '@context':
```

Comment:
How should I describe all these keys (with `@` symbols), I'm assuming this is a part of the JSON-LD structure.